xlive/Blam/Engine/tag_files/tag_loader/tag_injection_manager.cpp

```diff
@@ -0,0 +1,688 @@
+#include "stdafx.h"
+#include "tag_injection_manager.h"
+
+#include "bitmaps/bitmap_group.h"
+#include "cache/pc_geometry_cache.h"
+#include "cache/pc_texture_cache.h"
+#include "creatures/creature_definitions.h"
+#include "geometry/geometry_block.h"
+#include "models/render_model_definitions.h"
+#include "physics/physics_model_definitions.h"
+#include "render/weather_definitions.h"
+#include "units/biped_definitions.h"
+#include "units/vehicle_definitions.h"
+#include "Util/filesys.h"
+
+#define lazy_fread(FILE, OFFSET, OUT, SIZE, COUNT)\
+	fseek(FILE, OFFSET, SEEK_SET);\
```

Comment:
wonder what would happen if I were to invoke this macro inside a brace-less if statement scope.